welcome/index.adoc

```diff
@@ -72,15 +72,18 @@ Explore the following {product-title} installation tasks:
 
 - **xref:../installing/index.adoc#ocp-installation-overview[{product-title} installation overview]**: Depending on the platform, you can install {product-title} on installer-provisioned or user-provisioned infrastructure. The {product-title} installation program provides the flexibility to deploy {product-title} on a range of different platforms.
 
+// PR open https://github.com/openshift/openshift-docs/pull/77474
```

Comment:
Awaiting for the feature docs to be live. See https://issues.redhat.com/browse/OSDOCS-10687

Reply:
This issue is at risk and will probably not be ready in time. I recommended that the PR to remove agent-based installation of Alibaba Cloud be reverted if  possible. I guess we will hear the decision tomorrow.